ai-ml/jark-stack/preload-container-image-ami/README.md

```diff
@@ -0,0 +1,64 @@
+# Preload Container Images 
```

Comment:
Let's kill this doc, you are documenting it in the website which is good enough.

Reply:
No problem